packages/vite-plugin-cloudflare/playground/dynamic-import-paths/__tests__/dynamic-import-paths.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { expect, test } from "vitest";
+import { getJsonResponse, isBuild } from "../../__test-utils__";
+
+test.runIf(isBuild)(
```

Comment:
I couldn't get this test to pass in dev on Windows and I'm not sure why. Seems like it could possibly be a bug in Vite - https://github.com/vitejs/vite/issues/15258.